pandas/core/reshape/merge.py

```diff
@@ -744,6 +747,57 @@ def get_result(self) -> DataFrame:
 
         return result.__finalize__(self, method="merge")
 
+    def _anti_join_update(self):
+        def _anti_helper(_left, _right, _how):
```

Comment:
move this helper to its own module level function & fully type in & out arguments & add doc-strings